vuepress/docs/next/tutorials/solution/entando-hub.md

```diff
@@ -5,7 +5,7 @@ sidebarDepth: 2
 # Entando Hub
 ## Overview
 
-The enterprise Entando Hub enables teams to share components across the organization and between Entando Applications. It can be installed in Entando 6.3.2 or higher and includes API-level integration with the Entando 7.0 App Builder.
+The Entando Hub enables teams to share components across the organization and between Entando Applications. It can be installed in Entando 6.3.2 or higher and includes API-level integration with the Entando 7.0 App Builder.
```

Comment:
"the organization" is unclarified; which org is this referring to? any org or one in particular? (i.e. "their" org vs "an" org)